src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -6,7 +6,7 @@ lang: en
 
 # How to use a wallet
 
-Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our [How to: "Register" an Ethereum account](/guides/how-to-register-an-ethereum-account/).
+Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our [How to: create an Ethereum account](/guides/how-to-create-an-ethereum-account/).
```

Comment:
(We may want to add a redirect from the old link to the new one if we're changing the file/path name)